tests/e2e/domainRoute.spec.ts

```diff
@@ -37,19 +37,15 @@ describe("Get all transfers with a specific domain as source or destination", fu
     }
   })
 
-  it("Should successfully fetch all transfers from domain 1", async () => {
+  it("Should successfully fetch all domain 1 transfers", async () => {
     const res = await axios.get(`http://localhost:8000/api/domains/${DOMAIN_1}/transfers?page=1&limit=100`)
     const transfers = res.data as Array<TransferResponse>
 
     expect(res.status).to.be.deep.equal(200)
-    expect(transfers.length).to.be.deep.equal(32)
-
-    for (const transfer of transfers) {
-      expect(transfer.fromDomainId).to.be.deep.equal(parseInt(DOMAIN_1))
-    }
+    expect(transfers.length).to.be.deep.equal(35)
   })
 
-  it("Should successfully fetch all transfers from domain 1 with specified source", async () => {
+  it("Should successfully fetch all transfers from domain 1", async () => {
```

Comment:
Add tests for fetching transfers that are going to that domain

Reply:
For domain 1 that test already exists. Alternatively, I can add from/to/all transfer tests for every domain (additional tests for 2 and 3)?

Reply:
I guess it is fine then.
Though additional tests would be nice.